swagger.json

```diff
@@ -5200,64 +5200,115 @@
+                    "required": [
+                        "driverId",
+                        "driverCreatedAtMs",
+                        "documentType",
+                        "fields"
+                    ],
+                    "properties": {
+                        "driverId": {
+                            "type": "integer",
+                            "format": "int64",
+                            "description": "ID of the driver that submitted this document.",
+                            "example": 555
+                        },
+                        "driverCreatedAtMs": {
+                            "type": "integer",
+                            "format": "int64",
+                            "description": "The time in Unix epoch milliseconds that the document was created by the driver.",
+                            "example": 1462881998034
+                        },
+                        "documentType": {
+                            "type": "string",
+                            "description": "Descriptive name of this type of document.",
+                            "example": "Fuel Receipt"
+                        },
+                        "vehicleId": {
+                            "type": "integer",
+                            "format": "int64",
+                            "description": "VehicleID of the driver at document creation.",
+                            "example": 222
+                        },
+                        "fields": {
+                            "description": "The fields associated with this document.",
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/DocumentFields"
+                            }
+                        }
+                    }
+                },
+                { "$ref": "#/definitions/DocumentBase"}
+            ]
+        },
+        "DocumentCreate": {
+            "allOf": [
+                {
+                    "type": "object",
+                    "description": "Arguments to create a document.",
+                    "required": [
+                        "documentTypeUuid",
+                        "fields"
+                    ],
+                    "properties": {
+                        "documentTypeUuid": {
+                            "type": "string",
+                            "description": "Universally unique identifier for the document type this document is being created for.",
+                            "example": "4aff772c-a7bb-45e6-8e41-6a53e34feb83"
+                        },
+                        "fields": {
+                            "description": "List of fields should match the document type’s list of field types in the correct order. In other words, a field's valueType and value (aka only one of: stringValue, numberValue, or photoValue) at index i should match with the document field type’s valueType at index i.",
```

Comment:
nit: "aka" -> "i.e."? Also, should we italicize the "i" used for index?

Reply:
Sure. Not sure how to italicize but I can try to figure it out.